bukkit/src/main/java/net/skinsrestorer/bukkit/SkinApplierBukkit.java

```diff
@@ -215,4 +216,26 @@ private void checkOptFile() {
         log.debug("[Debug] Opt Files: { disableDismountPlayer: " + disableDismountPlayer + ", disableRemountPlayer: " + disableRemountPlayer + ", enableDismountEntities: " + enableDismountEntities + " }");
         optFileChecked = true;
     }
+
+    private boolean isPaper() {
+        if (PaperLib.isPaper() && ReflectionUtil.SERVER_VERSION.isNewer(new ServerVersion(1, 11, 2))) {
+            if (hasPaperMethods()) {
+                return true;
+            } else {
+                log.warning("Paper detected, but the methods are missing. Disabling Paper Refresher.");
```

Comment:
This should be a debug, as this is not a warning that players should report to us or that will effect the status of the working condition of the plugin (as it will fall back to spigot refresher)